apps/dotcms-ui/src/app/portlets/shared/dot-content-types-listing/dot-content-types.component.ts

```diff
@@ -121,6 +133,14 @@ export class DotContentTypesPortletComponent implements OnInit {
         this.listing.loadFirstPage();
     }
 
+    public hideCloneContentTypeDialog() {
```

Comment:
no need to add `public`, also add doc

Reply:
Done. And removed the public, and added doc to `saveCloneContentTypeDialog` too